proto/expr.proto

```diff
@@ -271,6 +271,8 @@ message ExprNode {
     // Adminitration functions
     COL_DESCRIPTION = 2100;
     CAST_REGCLASS = 2101;
+    // System information functions
+    PG_GET_INDEXDEF = 2400;
```

Comment:
https://www.postgresql.org/docs/16/functions-info.html#FUNCTIONS-INFO-CATALOG

Reply:
I think [system info functions](https://www.postgresql.org/docs/16/functions-info.html) and [system admin functions](https://www.postgresql.org/docs/16/functions-admin.html) are two different sets of functions. Let's keep them separated.

Reply:
However, `COL_DESCRIPTION` should be `system info function` instead of `system admin function`.🤪

Reply:
Moved `COL_DESCRIPTION` to `system info function` category. Since all these functions are local-execution-mode-only, it's OK to change the tag numbers.

Reply:
You can't break protobuf backward compalibility

Reply:
It's OK, with our current design, these function calls will never be persisted.